cockroachcloud/quickstart-trial-cluster.md

```diff
@@ -0,0 +1,183 @@
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ sudo cp -i cockroach-{{ page.release_info.version }}.linux-amd64/cockroach /usr/local/bin/
+    ~~~
+    </section>
+
+3. Initialize the `movr` workload using the `cockroach workload` command and the [connection string](#step-4-get-the-connection-string).
+
+    In the connection string, the SQL user's username is prepopulated. Replace `<password>` with the SQL user's password that you entered in [Step 2](#step-2-create-a-sql-user). Replace the `<certs_dir>` placeholder with the path to the `certs` directory that you created in [Step 4](#step-4-get-the-connection-string).
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach workload init movr \
+    'postgres://<username>:<password>@<global host>:26257/movr?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    ~~~
+
+4. Use the built-in SQL client to view the database:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach sql \
+    --url='postgres://<username>:<password>@<global host>:26257/movr?sslmode=verify-full&sslrootcert=<certs_dir>/<ca.crt>'
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SHOW TABLES FROM movr;
+    ~~~
+
+    ~~~
+                  table_name
+    ------------------------------
+      promo_codes
+      rides
+      user_promo_codes
+      users
+      vehicle_location_histories
+      vehicles
+    (6 rows)
+    ~~~
+
+5. Run your first query:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SELECT * FROM movr.users WHERE city='new york';
+    ~~~
+
+    ~~~
+                           id                  |   city   |       name       |           address           | credit_card
+    ---------------------------------------+----------+------------------+-----------------------------+--------------
+      00000000-0000-4000-8000-000000000000 | new york | Robert Murphy    | 99176 Anderson Mills        | 8885705228
+      051eb851-eb85-4ec0-8000-000000000001 | new york | James Hamilton   | 73488 Sydney Ports Suite 57 | 8340905892
+      0a3d70a3-d70a-4d80-8000-000000000002 | new york | Judy White       | 18580 Rosario Ville Apt. 61 | 2597958636
+      0f5c28f5-c28f-4c00-8000-000000000003 | new york | Devin Jordan     | 81127 Angela Ferry Apt. 8   | 5614075234
+      147ae147-ae14-4b00-8000-000000000004 | new york | Catherine Nelson | 1149 Lee Alley              | 0792553487
+      19999999-9999-4a00-8000-000000000005 | new york | Nicole Mcmahon   | 11540 Patton Extensions     | 0303726947
+    (6 rows)
+    ~~~
+
+## What's next?
+
+Learn more:
+
+- Use the [built-in SQL client](connect-to-your-cluster.html#use-the-cockroachdb-sql-client) to connect to your cluster and [learn CockroachDB SQL](learn-cockroachdb-sql.html).
+- Build a ["Hello World" app with the Django framework](build-a-python-app-with-cockroachdb-django.html), or [install another client framework](../stable/install-client-drivers.html) for your preferred language.
```

Comment:
the second link might be clearer as "install a client driver .. for your fave language" since IIRC not everything on that linked page is a framework